### PR TITLE
fix assembly reference

### PR DESCRIPTION
**Pull request by dsyme**
@brettfo Maybe we can just delete these special entries for ``System.Collections.Immutable`` now

### CODE REVIEW

File: vsintegration/Vsix/VisualFSharpOpenSource/VisualFSharpOpenSource.csproj [hunk 1]
```diff
@@ -324,8 +324,8 @@
       <SpecificVersion>False</SpecificVersion>
       <HintPath>$(FSharpSourcesRoot)\..\packages\Newtonsoft.Json.10.0.2\lib\net45\Newtonsoft.Json.dll</HintPath>
     </Reference>
-    <Reference Include="System.Collections.Immutable, Version=1.3.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
-      <HintPath>$(FSharpSourcesRoot)\..\packages\System.Collections.Immutable.$(SystemCollectionsImmutableVersion)\lib\portable-net45+win8+wp8+wpa81\System.Collections.Immutable.dll</HintPath>
+    <Reference Include="System.Collections.Immutable, Version=1.2.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a, processorArchitecture=MSIL">
```
Comment (KevinRansom) [inline vsintegration/Vsix/VisualFSharpOpenSource/VisualFSharpOpenSource.csproj]: Is assembly version 1.3.1 really in package 1.2.1?  gosh that seems idiosyncratic.

Comment (dsyme) [reply vsintegration/Vsix/VisualFSharpOpenSource/VisualFSharpOpenSource.csproj]: Yes

Comment (dsyme) [reply vsintegration/Vsix/VisualFSharpOpenSource/VisualFSharpOpenSource.csproj]: Discovered that the hard way

Comment (dsyme) [reply vsintegration/Vsix/VisualFSharpOpenSource/VisualFSharpOpenSource.csproj]: But hey, think what people say about FSharp.Core...

Comment (KevinRansom) [reply vsintegration/Vsix/VisualFSharpOpenSource/VisualFSharpOpenSource.csproj]: Well .. that sure is a thing ...

**Comment by abelbraaksma** [comment]
Is this fix related to #3629? Note that that issue is about 166 such warnings in current master, and not only about Immutable.

**Comment by KevinRansom** [comment]
@abelbraaksma I think this is a small correction to an earlier brettfo PR.

**Comment by abelbraaksma** [comment]
Ok, so unrelated then ;).

**Pull request merged**